test/core/extensions/coreIssuanceOrder.spec.ts

```diff
@@ -156,6 +156,15 @@ contract("CoreIssuanceOrder", (accounts) => {
       assertTokenBalance(setToken, existingBalance.add(subjectQuantityToIssue), signerAddress);
     });
 
+    it("marks the correct amount as filled in orderFills mapping", async () => {
```

Comment:
Make sure your test pulls the value before (even if it's 0), and checks that the ending value is what you intended. See: `coreIssuance.spec.ts#transfers the required tokens from the user`

Reply:
Cool